### DISCUSSION TITLE
[Bug]: 無法擷取遊戲畫面,且一直顯示"尝试启动游戏时发生错误：切换到游戏超时"

### DISCUSSION THREAD

**Discussion by Mamgo1128** [moesnow/March7thAssistant#241] [Q&A] [answered]
### 请确保您已阅读以上注意事项，并勾选下方的确认框。

- [X] 我已经使用一个干净且无其它非必要的插件的环境测试过，问题依旧存在。
- [X] 我已经在 [Issue Tracker](https://github.com/moesnow/March7thAssistant/issues) 中找过我要提出的问题，没有找到相同问题的ISSUE。
- [X] 我已知晓并同意，若这个 Issue 是关于其他非程序本身问题，则我的 Issue 可能会被无条件自动关闭/锁定。
- [X] 我已知晓并同意，如果我不按照模板提供的指示进行填写，则我的 Issue 可能会被无条件删除。

### March7thAssistant 版本 （或 Commit Id）

2.1.1.1

### 重现步骤

點擊完整執行時

### 期望的结果是什么？

修復

### 实际的结果是什么？

一直重開遊戲 且無法使用所有功能

**Comment by Mamgo1128** [comment]
![image](https://github.com/moesnow/March7thAssistant/assets/124562166/c1a09e24-895a-425a-acd1-b77e8e51b9b6)

**Comment by moesnow** [comment, marked as answer]
游戏语言只支持简体中文

**Comment by Mamgo1128** [comment]
太感謝你了 他成功運行了
